conftest.py

```diff
@@ -66,3 +67,8 @@ def professional(profile3):
 def chatmessage(professional, client):
     return Chatmessage(professional_id=professional,
                        client_id=client, message="message1", sender_type=SenderType.Client)
+
+
+@pytest.fixture
+def searchHistory(professional, client):
+    return SearchHistory(professional_id=professional, client_id=client, date=datetime(2023, 4, 17, 10, 0, 0))
```

Comment:
Try to use relative times instead, like (now - 5 days). Decreases maintenance in tests' dates

Reply:
My mistake, the date is defaulted to be now

Reply:
Why did you delete the date?

Reply:
Because we need the default time and it is the current time that was at the time the object was created